whitepaper/chapters/consensus.tex

```diff
@@ -38,7 +38,7 @@ \section{Consensus}
 Proof of Stake (PoS)\cite{King:2012:PPP}\cite{Nxt2013} blockchains were introduced after Bitcoin.
 They presented an alternative solution to the Byzantine consensus problem that did not require significant power consumption.
 Fundamentally, these chains behaved similarly to Bitcoin with one important difference.
-Instead of predicating the probability of creating a block on a node's relative hash rate the probability is based on a node's relative stake in the network.
+Instead of predicting the probability of creating a block on a node's relative hash rate, the probability is based on a node's relative stake in the network.
```

Comment:
this changes meaning. if you don't like "predicating" consider "basing" or "deriving".